packages/@adobe/spectrum-css-temp/components/inputgroup/skin.css

```diff
@@ -34,6 +34,33 @@ governing permissions and limitations under the License.
       }
     }
   }
+
+  &.spectrum-InputGroup--menuOpen {
+    .spectrum-FieldButton {
+      background-color: var(--spectrum-global-color-gray-200);
+      border-color: var(--spectrum-global-color-gray-400);
+
+      .spectrum-Dropdown-chevron {
+        color: var(--spectrum-global-color-gray-700);
+      }
+    }
+
+    &:hover {
+      &:not(.is-invalid):not(.is-disabled) {
+        &:not(.is-focused){
+          .spectrum-FieldButton {
+            border-color: var(--spectrum-global-color-gray-500);
+          }
+        }
+
+        .spectrum-FieldButton {
+          &:hover .spectrum-Dropdown-chevron {
+            color: var(--spectrum-global-color-gray-900);
+          }
```

Comment:
For the `The button should retain the "down" style while the menu is open` design feedback